codespell_lib/data/dictionary.txt

```diff
@@ -25633,16 +25739,25 @@ opulate->populate, opiate, opulent,
 opulates->populates, opiates,
 opyion->option
 opyions->options
+orangatang->orangutan
+orangatangs->orangutans
 orcale->oracle
+orcestrate->orchestrate
```

Comment:
Add `orcestra->orchestra` and plural `orcestras->orchestras`?

Reply:
Yes. Added.